src/tools/altinn-fleet-stats/backend/altinn_fleet/scanner.py

```
@@ -0,0 +1,576 @@
+        run = conn.execute(
+            "INSERT INTO scan_runs (started_at, status) VALUES (?, ?)",
+            (datetime.now(timezone.utc).isoformat(), "running"),
+        )
+        run_id = run.lastrowid
+        conn.commit()
+
+        for idx, app_dir in enumerate(app_dirs, 1):
+            try:
+                status = scan_app(conn, app_dir, settings.env, force=force)
+                conn.commit()
+            except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)